subt/docker/robotika/subt_seed_node.cc

```diff
@@ -0,0 +1,508 @@
+/*
```

Comment:
Why is this file here? It has the contents of (some version of) `ros_proxy_node.cc` but I don't see it being used for anything.

Reply:
Probably it is not need it. I will test it.